hooks.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I really liked this name "hooks". This way we don't need to worry about a misunderstanding between this and the dispatched. However, I didn't get why we added this "path" here. Is there any situation where we can send to different paths?

Reply:
Path is there just because I was lazy and did not want to parse the URL in two pieces: domain and path.
That's because Node.js wants you to pass that as two separate properties.